src/main/java/com/imsweb/mph/MphUtils.java

```diff
@@ -72,55 +72,34 @@ public enum MpResult {
         INVALID_INPUT
     }
 
-    // the unique instance of this utility class
-    private static MphUtils _INSTANCE = null;
```

Comment:
I am confused by this.   You are removing the instance? Are you expecting every software to now cache they own instance? Except for SEER*API, no software need to support a provider.   Right now SEER*Abs does something like "MphUtils.getInstance().xxx()".   Are you expecting that software to have to do "new MphUtils().xxx()" and create a new instance every time? Or to cache it itself? Neither of those options look good.  Why did we have to remove the instance?